tests/test_dataclasses.py

```diff
@@ -33,7 +33,11 @@ def test_json_datetime_serializer():
     assert json_datetime_serializer("") is None
 
     my_date_time = DateTime.today()
-    assert json_datetime_serializer(my_date_time) == my_date_time.isoformat()
+    assert json_datetime_serializer(my_date_time) == datetime_to_string_incl_seconds(my_date_time)
+
+
+def test_time_stamp() -> DateTime:
```

Comment:
The name of this function gives the impression that this is a test function, I suggest renaming it to something else, extra points if you set it as a constant instead of a function

Reply:
Indeed, I can not remember why I created a function for it. Will change to a constant and rename. Thanks!